package.json

```diff
@@ -48,6 +48,8 @@
   },
   "dependencies": {
     "base-64": "^0.1.0",
```

Comment:
Do you think we can migrate to https://github.com/hapijs/b64 instead? 
It doesn't have to be this particular PR, but using a better maintained family of libraries is preferred.